pkgs/top-level/python-packages.nix

```diff
@@ -1102,6 +1102,20 @@ in {
 
   cheroot = callPackage ../development/python-modules/cheroot {};
 
+  cli-helpers = buildPythonPackage rec {
```

Comment:
This needs to be put in `pkgs/development/python-modules/cli-helpers/default.nix`. The expression should adhere to the newest standards (`pname` etc.) and adding `cli-helpers` should be a separate commit.